Makefile

```diff
@@ -5,6 +5,9 @@ DOCKERBUILD := docker build --build-arg http_proxy=$$http_proxy --build-arg http
 
 PIP?=pip
 
+VENV_NAME:=.venv$(VENV_PY_VERSION)
```

Comment:
Hello!
Sorry for delay please.
I have tried pipenv. It is very handy but I had two problems.
1) Have you written Pipfile for buildbot development? 
How to specify something like 
pip install -e 'master[tls,test,docs]'  ?
I haven't found example in doc.
Also I tried to install it using pip and then generate Pipefile using pipenv. 
But I haven't found how to do it.

2) Venv folder is the same for every python version.
For example when I switched from py2 to py3, py2 venv was removed.
So each time you wanna switch to another py version fresh venv will be created again with all
dependencies. It takes some time. You cannot simply activate one of venvs created earlier.